Classes/ActionButton/ActionButton.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  ActionButton.swift
+//  AutoEscuelaComponents
+//
+//  Created by user159106 on 7/7/20.
+//
+
+import UIKit
+import Reusable
+
+public class ActionButton: UIView, NibOwnerLoadable{
+
+    @IBOutlet var button: UIButton!
+    /*
+    // Only override draw() if you perform custom drawing.
+    // An empty implementation adversely affects performance during animation.
+    override func draw(_ rect: CGRect) {
+        // Drawing code
+    }
+    */
+    var action : (() -> Void)?
+    
+    public init() {
+        super.init(frame: CGRect.zero)
+        self.loadNibContent()
+    }
+    
+    public override init(frame: CGRect) {
+        super.init(frame: frame)
+        self.loadNibContent()
+    }
+    
+    public required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        self.loadNibContent()
+    }
+   
+    public func configure(title: String, action: (() -> Void)?) {
+        self.button.titleLabel?.textColor = .white
+        self.button.setTitle(title, for: .normal)
+        self.button.backgroundColor = UIColor(hex: "#6D58FF")
```

Comment:
In a future, if you keep using hexadecimal colours, it will be nice to have an external class, that gives the colour without knowing which hexadecimal is, like a static class "Colour".